src/lib/components/Widget.tsx

```diff
@@ -64,6 +67,15 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
   }
 `
 
+function Updater() {
```

Comment:
Nit - can we changer to `Updaters`

Reply:
lol my personal nit was to change from `Updaters` to singular. I'll use the plural 😆